auth/src/main/java/com/firebase/ui/auth/ui/email/RegisterEmailActivity.java

```diff
@@ -111,28 +149,156 @@ private void setUpTermsOfService() {
         agreementText.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View view) {
-                Intent intent = new Intent(Intent.ACTION_VIEW).setData(Uri.parse
-                        (mActivityHelper.getFlowParams().termsOfServiceUrl));
-                startActivity(intent);
+                // Getting default color
+                TypedValue typedValue = new TypedValue();
+                getTheme().resolveAttribute(R.attr.colorPrimary, typedValue, true);
```

Comment:
[This works](https://youtu.be/SmCo8-371Ec), but what if they don't have a `colorPrimary`. (Actually is it even possible to not have a colorPrimary? I'm pretty sure `AppCompatActivity` uses that for all default theming)